.github/workflows/test_frameworks.yml

```diff
@@ -278,3 +278,32 @@ jobs:
         run: "make init"
       - name: Run tests
         run: ddtrace-run pytest -p no:warnings tests
+
+  asyncpg-testsuite-0_25_0:
+    runs-on: "ubuntu-latest"
+    env:
+      DD_TESTING_RAISE: true
+      DD_PROFILING_ENABLED: true
+    defaults:
+      run:
+        working-directory: asyncpg
+    steps:
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+      - uses: actions/checkout@v3
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v3
+        with:
+          repository: magicstack/asyncpg
+          ref: v0.25.0
+          path: asyncpg
+      - name: Install ddtrace
+        run: pip install ../ddtrace
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip setuptools wheel
```

Comment:
surprised you explicitly need `setuptools` and `wheel` even after upgrading to a modern `pip`.

Reply:
yeah not sure why they do it 🤷 https://github.com/MagicStack/asyncpg/blob/v0.25.0/.github/workflows/tests.yml#L125